echopype/convert/convert.py

```diff
@@ -24,15 +24,23 @@ def Convert(path='', xml_path=''):
         Specialized convert object that will be used to produce a .nc file
     """
 
-    file_name = os.path.basename(path)
-
     if path:
-        if not os.path.isfile(path):
-            raise FileNotFoundError(f"There is no file named {os.path.basename(path)}")
+        if isinstance(path, list):
+            file_name = os.path.basename(path[0])
+            ext = os.path.splitext(file_name)[1]
+            for p in path:
+                if not os.path.isfile(p):
+                    raise FileNotFoundError(f"There is no file named {os.path.basename(path)}")
+                if os.path.splitext(p)[1] != ext:
+                    raise ValueError("All files are not in the same format")
```

Comment:
This seems a weird sentence, better phrased as "Not all files are in the same format."